docs/tre-admins/environment-variables.md

```diff
@@ -36,3 +36,4 @@
 | `RESOURCE_PROCESSOR_VMSS_SKU` | The SKU of the VMMS to use for the resource processing VM. |
 | `CORE_APP_SERVICE_PLAN_SKU` | The SKU of AppService plans created for the core infrastructure. |
 | `WORKSPACE_APP_SERVICE_PLAN_SKU` | Optional. The SKU used for AppService plan used in E2E tests unless otherwise specified. Default value is `P1v2`. |
+| `RESOURCE_PROCESSOR_NUMBER_PROCESSES_PER_INSTANCE` | Optional. The number of processes to create in the Resource Processor. Equates to the number of parallel deployment operations possible in your TRE. Defaults to `2`. |
```

Comment:
5?

This needs to be documented [here ](https://microsoft.github.io/AzureTRE/tre-admins/setup-instructions/cicd-pre-deployment-steps/#configure-core-secrets) as well as CICD setup docs don't reference this md file.